README.md

```diff
@@ -6,91 +6,121 @@ npm install ethfs-uploader
+| Optimism                   | oeth / 10                     | 
+| Optimism Testnet           | ogor / 420                    | 
+| Arbitrum One               | arb1 / 42161                  | 
+| Arbitrum Nova              | arb-nova / 42170              | 
+| Arbitrum Testnet           | arb-goerli / 421613           | 
+| Web3Q Galileo Testnet      | w3q-g / 3334                  | 
+| BNB Smart Chain            | bnb / 56                      | 
+| BNB Smart Chain Testnet    | bnbt / 97                     | 
+| Avalanche C-Chain          | avax / 43114                  | 
+| Avalanche Fuji Testnet     | fuji / 43113                  | 
+| Fantom Opera               | ftm / 250                     | 
+| Fantom Testnet             | tftm / 4002                   | 
+| Polygon Mainnet            | matic / 137                   | 
+| Polygon Mumbai             | maticmum / 80001              | 
+| Polygon zkEVM Testnet      | zkevmtest / 1402              | 
+| QuarkChain Mainnet Shard 0 | qkc-s0 / 100001               |
+| QuarkChain Devnet Shard 0  | qkc-d-s0 / 110001             |
+| Harmony Mainnet Shard 0    | hmy-s0 / 1666600000           |
+| Harmony Testnet Shard 0    | hmy-b-s0 / 1666700000         |
+| Evmos                      | evmos / 9001                  | 
+| Evmos Testnet              | evmos-testnet / 9000          |
+ 
+
 ## Usage
 ### Support EIP-3770 Address
 ```
-mainnet 
-    w3q:<domain/address>
-
-galileo
-    w3q-g:<domain/address>
-
 ethereum
-    eth:<domain/address>
+    eth:<domain|address>
 
-rinkeby
-    rin:<domain/address>
+... 
+
+galileo
+    w3q-g:<domain|address>       
 ```
 ##### Example
 ```
-mainnet
-    w3q:home.w3q
-
-galileo
-    w3q-g:0x1825...2388
-
 ethereum
     eth:ens.eth
 
-rinkeby
-    rin:ens.eth
+...
+
+galileo
+    w3q-g:0x1825...2388
 ```
 <br/>
 
 
 
 ### Create FlatDirectory Command
+FlatDirectory is created by default on the Galileo network. You can specify to create a network by setting "chainId". If it is Ethereum or an unsupported network, RPC needs to be set.
 ```
 npx ethfs-uploader --create --privateKey <private-key>
+npx ethfs-uploader --create --privateKey <private-key> --chainId <chainId>
+npx ethfs-uploader --create --privateKey <private-key> --chainId <chainId> --RPC <rpc>
 
 // output: contract address 
 ```
 ##### Example
 ```
 npx ethfs-uploader --create --privateKey 0x32...
+npx ethfs-uploader --create --privateKey 0x32... --chainId 5
+npx ethfs-uploader --create --privateKey 0x32... --chainId 1 --RPC https://rpc.ankr.com/eth
 ```
 <br/>
 
 
 
 ### Deploy Command
 ```
-w3ns
-    npx ethfs-uploader <directory/file> <domain/address> --privateKey <private-key>
-
 ens
-    npx ethfs-uploader <directory/file> <domain> --privateKey <private-key> --RPC <rpc-url>
+    npx ethfs-uploader <directory|file> <domain> --privateKey <private-key> --RPC <rpc-url>
+w3ns
+    npx ethfs-uploader <directory|file> <domain> --privateKey <private-key>
+other networks
```

Comment:
Not "other networks" But should be "FlatDirectory address" when comparing to ens/w3ns